config/config/sc-config.yaml

```diff
@@ -238,6 +238,7 @@ prometheus:
 
   # Todo remove dependencie on alertmanager from service cluster
   alertmanagerSpec:
+    replicas: 2
```

Comment:
Now I'm confused. Didn't we need 3 replicas?

Reply:
Turned out we didnt! :+1: